lib/ruby_smb/smb1/bit_field/create_options.rb

```diff
@@ -27,12 +28,12 @@ class CreateOptions < BinData::Record
         # Byte Boundary
         bit1    :open_for_free_space_query,   label: 'Open for Free Space Query'
         bit1    :open_no_recall,              label: 'Open No Recall'
-        bit1    :reserved2,                   label: 'Reserved Space'
+        bit1    :open_reparse_point,          label: 'Open Reparse Point'
```

Comment:
I added the FILE_OPEN_REPARSE_POINT field according to [2.2.4.9.1 Client Request Extensions] (https://msdn.microsoft.com/en-us/library/cc246332.aspx).